src/utils/settings.ts

```diff
@@ -29,28 +40,66 @@ export class SettingsStatic {
     )
   }
 
+  public static settingsFileType(path) {
+    const files: string[] = fs.readdirSync(path)
+    const filteredFiles = files ? files.filter(fn => fn.startsWith('settings')) : []
+    if (filteredFiles.length) {
+      const extension = extname(filteredFiles.pop())
+      return FileType[extension]
+    } else {
+      return null
+    }
+  }
+
+
+  public static loadSettings(path: string, fileType) {
+    debug('loading settings from %s', path)
+
+    switch (fileType) {
+      case FileType.json: {
+        debug('settings.json is deprecated, please use a yaml file based on resources/default-settings.yaml')
```

Comment:
this should be `console.warn()` because `debug` doesn't log unless explicitly enabled.. we should get rid of `debug` soon